src/main/java/ChatServer.java

```diff
@@ -0,0 +1,71 @@
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class ChatServer {
+    private ServerSocket serverSocket;
+    private UserIO io;
+
+    public ChatServer(ServerSocket serverSocket, UserIO io) {
+        this.serverSocket = serverSocket;
+        this.io = io;
+    }
+
+    public void readInFromClient() {
+        Socket server = makeConnection(serverSocket);
+        BufferedReader reader = createBufferedReader(server);
+        readInputTillOver(reader, server);
```

Comment:
A till is a little drawer for money, also something that is done to land to prepare it for crops